dags/destinations/gads_oci.py

```diff
@@ -37,12 +37,14 @@
   "gclid",
   "gbraid",
   "wbraid",
+  "currency_code",
```

Comment:
I think this will cause issues because of the line below at around 154 https://github.com/google-marketing-solutions/Tightlock/blob/logbot/dags/destinations/gads_oci.py#L154